saltstack/base/salt/telemetry/init.sls

```diff
@@ -1,13 +1,31 @@
 ## see more at internal repo: thunderhead/cdp-telemetry-cli
 {% set os = salt['environ.get']('OS') %}
-{% set cdp_telemetry_rpm_repo_url = salt['environ.get']('CDP_TELEMETRY_RPM_URL') %}
 {% set include_cdp_telemetry = salt['environ.get']('INCLUDE_CDP_TELEMETRY') %}
```

Comment:
there are lots of OS based conditionals in this file... it might be better to organize the logic in a separate sls for Centos and for others..